src/libs/nav.ts

```diff
@@ -73,13 +73,19 @@ export const getPath = (name: string, params?: Record<string, any>): string => {
 /**
  * alias for getPath()
  */
-export const getLink = (name: string, params?: Record<string, any>) => `#${getPath(name, params).slice(1)}`; // slice off leading slash
+export const getLink = (name: string, params?: Record<string, any>) =>
+  `#${getPath(name, params).slice(1)}${
```

Comment:
It might read a little clearer if this was split this into multiple lines instead of all in one format string.

Maybe something like:
```ts
const path = ...
const query = queryParams ? ... : ''
return `${path}${query}`
```